src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7051,400 +7247,19 @@ namespace UnitarySystems {
                 } else {
                     state.dataUnitarySystems->unitarySys[sysNum] = thisSys;
                 }
+            }
+        }
+        if (numAllSystemTypes == state.dataUnitarySystems->numUnitarySystems && state.dataZoneEquip->ZoneEquipInputsFilled)
+            setupAllOutputVars(state, numAllSystemTypes);
+    }
```

Comment:
Call setupAllOutputVars to set up report variables. This might not be the best place for this call but it seems to work for now.